app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, source_name, destination_name = command.split()
```

Comment:
before unpacking, you should check that the length of `command.split()` is three